src/dist/dist_cluster.cc

```diff
@@ -117,6 +118,11 @@ int main(int argc, char* argv[]) {
       "Recommended value 900s"
       " [0 (off)]",
       {'c', "checkpoint_interval"});
+  args::ValueFlag<std::string> file_name(
```

Comment:
use a bit more descriptive name, `file_name` makes following code unclear as to what file its dealing with.